src/pages/RestaurantInformation/RestaurantInformation.tsx

```diff
@@ -0,0 +1,80 @@
+import 'react-toastify/dist/ReactToastify.css';
+import informationAsset from '../../assets/images/restaurant/restaurant-details/information.svg';
+
+import pizza from '../../assets/images/restaurant/restaurant-menu/pizza.png';
+import hamburger from '../../assets/images/restaurant/restaurant-menu/hamburgers.png';
+import pancake from '../../assets/images/restaurant/restaurant-menu/pancakes.png';
+import IRestaurantDetails from '../../interfaces/IRestaurantDetails';
+import Menu from './Menu';
+import IMenu from '../../interfaces/IMenu';
+import Slider from './Slider';
+
+const RestaurantInformation = () => {
+  const restaurant: IRestaurantDetails = {
```

Comment:
same as above, we should move these somewhere, maybe they're gonna be re-used in this prototype so its better if they are outside this file and exported

Reply:
Solved